metro.config.js

```diff
@@ -3,10 +3,7 @@
  * integration tests during local development or on CI services.
  */
 
-const blacklist = require('metro-config/src/defaults/blacklist');
```

Comment:
Why are you removing all these?  -- I suspect the macos builds may still need them.

Reply:
Sync'd offline.